tkg/tkgconfigpaths/zz_bundled_default_bom_files_configdata.go

```diff
@@ -12,5 +12,5 @@ package tkgconfigpaths
 var (
 	TKGDefaultImageRepo               string = "projects-stg.registry.vmware.com/tkg"
 	TKGDefaultCompatibilityImagePath  string = "framework-zshippable/tkg-compatibility"
-	TKGManagementClusterPluginVersion string = "v0.28.0-dev"
+	TKGManagementClusterPluginVersion string = "v0.7.0-dev"
```

Comment:
question: why this change ?

Reply:
this one somehow slipped through, i reverted this change